src/styles/blocks/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background: hsl(0, 0%, 100%);
+  border: 1px solid hsl(0, 0%, 95%);
+  border-radius: 5px;
```

Comment:
Group styles, according to the rules - https://codeguide.academy/html-css.html#css-order